lib/CTownHandler.h

```diff
@@ -253,6 +253,21 @@ class DLL_LINKAGE CTown
 
 	template <typename Handler> void serialize(Handler &h, const int version)
 	{
+		if(version < 777 && !h.saving)
+		{
+			si32 moatDamage;
+			h & moatDamage;
+		}
+		if(version >= 758 && version < 777)
+		{
+			std::vector<BattleHex> moatHexes;
+			h & moatHexes;
+		}
+		else if(!h.saving)
+		{
+			std::vector<BattleHex> moatHexes = {11, 28, 44, 61, 77, 111, 129, 146, 164, 181};

```

Comment:
I do not understand this line

Reply:
hahah, just copy old code and don't see it :D